packages/tldraw/src/test/TestEditor.ts

```diff
@@ -131,6 +131,10 @@ export class TestEditor extends Editor {
 		})
 	}
 
+	getHistory() {
+		return this.history
+	}
+
```

Comment:
Now that Editor.history is protected, I've added a `getHistory` method for the test editor.